docs/docs/Configuration/configuration-api-keys.md

```diff
@@ -7,7 +7,14 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import Icon from "@site/src/components/icon";
 
-Langflow provides an API key functionality that allows users to access their individual components and flows without traditional login authentication.
+Langflow provides an API key functionality that allows users to access their individual components and flows.
+
+## Auto-login and API key authentication
+
+Prior to Langflow v1.5, when `AUTO_LOGIN` was enabled with `AUTO_LOGIN=true`, Langflow automatically logged users in as a superuser without requiring authentication, and API requests could be made without a Langflow API key.
+
+As of Langflow v1.5, all API requests require a Langflow API key, even when `AUTO_LOGIN` is enabled.
+The MCP-related endpoints will continue to require no authentication when `AUTO_LOGIN=true`.
```

Comment:
These two sentences contradict each other. Is it all endpoints or all-except-MCP? and which are the MCP endpoints?